pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *_test.py
-testpaths = tests
+testpaths = hanadb_exporter
```

Comment:
Why have you changed that? The tests are stored in `tests` folder

Reply:
That was a mistake that I don't know how it happened :o